app/src/main/java/br/com/dionataferraz/vendas/account/data/local/AccountDatabase.kt

```diff
@@ -0,0 +1,28 @@
+package br.com.dionataferraz.vendas.account.data.local
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import androidx.room.TypeConverters
+import br.com.dionataferraz.vendas.account.converters.Converters
+
+@Database(entities = [AccountEntity::class], version = 1)
```

Comment:
Vi que foi criado 3 databases, mas nós poderiamos utilizar apenas um e passar as outras entidades dessa forma:
```suggestion
            @Database(entities = [AccountEntity::class, UserEntity::class, TransactionEntity::class], version = 1)
```